utils/Bootstrap.ts

```diff
@@ -97,8 +97,7 @@ export async function scheduleBootstrap(
 ) {
   const api = getApi();
   const bootstrapBlockNumber = (await getBlockNumber()) + waitingPeriod;
-  return await signTx(
```

Comment:
why not using the regular signTx with ethkeypair?

Reply:
Fixed issues